ApplicationLibCode/ProjectDataModel/Summary/RimSummaryRegressionAnalysisCurve.cpp

```diff
@@ -382,3 +383,94 @@ QString RimSummaryRegressionAnalysisCurve::generateRegressionText( const regress
     // TODO: Display more parameters here.
     return "";
 }
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+void RimSummaryRegressionAnalysisCurve::appendTimeSteps( std::vector<time_t>& timeSteps, const std::set<QDateTime>& moreTimeSteps )
+{
+    for ( const QDateTime& t : moreTimeSteps )
+        timeSteps.push_back( RiaTimeTTools::fromQDateTime( t ) );
+}
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+std::vector<time_t> RimSummaryRegressionAnalysisCurve::getOutputTimeSteps( const std::vector<time_t>& timeSteps ) const
```

Comment:
Consider adding a unit test for this code

Reply:
Fixed.